meta-mbl-distro/recipes-extended/shadow/shadow_%.bbappend

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2019 Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: MIT
+
+FILESEXTRAPATHS_append := "${THISDIR}/files:"
+
+SRC_URI_append_class-native = " file://check-pass-min-len-for-clear-text-password.patch"
+
+MBL_MIN_PASS_LEN ?= "5"
+
+do_install_append () {
+
+    # We want to use SHA512 as encryption method to generate passwords
+    sed -i 's/^#ENCRYPT_METHOD.*$/ENCRYPT_METHOD SHA512/' ${D}${sysconfdir}/login.defs
+
+    # We want the minimum password length set by MBL_MIN_PASS_LEN
+    sed -i 's/^PASS_MIN_LEN.*$/PASS_MIN_LEN ${@d.getVar('MBL_MIN_PASS_LEN', True)}/' ${D}${sysconfdir}/login.defs
```

Comment:
Why use Python rather than just `${MBL_MIN_PASS_LEN}`?

Reply:
Addressed in 4a3571b.